pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/QuestionDao.java

```diff
@@ -1593,6 +1643,13 @@ default void insertQuestion(long activityId, AgreementQuestionDef agreementQuest
         }
     }
 
+    default boolean deleteDateQuestion(long questionId) {
+        getJdbiDateQuestionFieldOrder().deleteForQuestionId(questionId);
+        var val =  getJdbiDateQuestion().delete(questionId);
+        LOG.info("Deleted date question id? {}", val);
+        return val;
```

Comment:
Is there a need to clean up the resources that the question uses (templates, etc), or will the deletes cascade?

Reply:
I considered cleaning up and reached conclusion it was not worth the time and aggravation.